packages/components/src/internal/components/domainproperties/BaseDomainDesigner.tsx

```diff
@@ -161,17 +162,14 @@ export class BaseDomainDesigner extends PureComponent<BaseDomainDesignerProps> {
                         <Alert bsStyle="danger">{bottomErrorMsg}</Alert>
                     </div>
                 )}
-                <div className="domain-form-panel domain-designer-buttons">
-                    <Button onClick={onCancel}>Cancel</Button>
-                    <Button
-                        className="pull-right"
-                        bsStyle={successBsStyle || 'success'}
-                        disabled={submitting}
-                        onClick={onFinish}
-                    >
+                <FormButtons>
+                    <button className="btn btn-default" onClick={onCancel} type="button">
+                        Cancel
+                    </button>
+                    <button className={submitClassname} disabled={submitting} onClick={onFinish} type="button">
                         {saveBtnText}
-                    </Button>
-                </div>
+                    </button>
+                </FormButtons>
```

Comment:
not exactly sure where this feedback belongs in the PR but I pulled down the changes locally and looked at the sample type designer edit page and noticed that you can't quite scroll down far enough to see the last panel in all cases. not sure if we need some css update in the FormButtons component or if that will be up to each usage to provide enough bottom spacing.

<img width="1467" alt="Screenshot 2023-09-26 at 8 46 45 AM" src="https://github.com/LabKey/labkey-ui-components/assets/6411206/044ae9c9-1d9d-45e3-9d8d-6cf08ca1defe">

Reply:
I think the best way to resolve this is to just add padding to the bottom of our app body, I think we already had to do this for some other stuff, so it should just be a matter of increasing the amount of padding.